test/addons/make-callback-recurse/test.js

```diff
@@ -134,10 +134,10 @@ function checkDomains() {
     // Make sure nextTick, setImmediate and setTimeout can all recover properly
     // after a thrown makeCallback call.
     const d = domain.create();
-    d.on('error', common.mustCall(function(e) {
+    d.on('error', common.mustCall((e) => {
```

Comment:
Are extra arrow functions parentheses supposed to be used here?

If not:

```suggestion
    d.on('error', common.mustCall(e => {
```

Reply:
Please, leave the parentheses. Their use is enforced by our configuration on the `arrow-parens` rule in ESLint.

Reply:
Was just a question. Thanks, resolving!

Reply:
Yup, sorry if answered that a bit too matter-of-factly. 😀